.github/workflows/framework-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Framework
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  publish:
+    name: Publish draft
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Download wheel
```

Comment:
How can we ensure the job which actually creates and uploads the wheel is already finished? If I understand this  job right it would not create the wheel so the assumption is that the pipeline which run on the same commit is already finished right?

Reply:
@tanertopal I completely overlooked that indeed, thanks for the catch, I added a new step to wait for the upload to be finished